src/util/Permissions.js

```diff
@@ -57,7 +57,7 @@ class Permissions extends BitField {
  * * `ADD_REACTIONS` (add new reactions to messages)
  * * `VIEW_AUDIT_LOG`
  * * `PRIORITY_SPEAKER`
- * * `STREAM`
+ * * `VIDEO` (allows go live and video calls)
```

Comment:
You will need to also update the actual flag in the object below

Reply:
ah, missed that